virtual_robot:
  start_pose: [-0.6360336750246208, 0.1775516545663988, 0.2884846846489845, -0.16896014757275685, 1.2963893710523633, 0.9026939631222692 ]
  end_pose: [0.5737165893240899, -0.07601518975076951, 0.5510593852659934, 0.11929085613600919, 1.3173351072359232, 2.008197667675184 ]
  log_path: "/home/michaldobis/catkin_ws/datasets"
  attempts: 10 
